### PR TITLE
Fix flash CI

### PR DESCRIPTION
**Pull request by ethanwharris**
## Before submitting

- [x] Was this discussed/approved via a GitHub issue? (no need for typos and docs improvements)
- [x] Did you create/update your **configuration file**?
- [ ] Did you **set `runtimes` in config** for GitHub action integration?
- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ GitHub stars)?
- [ ] Are all integration **tests passing**?

## What does this PR do? \[optional\]

Fixes # (issue) \[add a link to the created issue in your repository if any\]
OR link your project for clarity.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: configs/PyTorchLightning/flash_pl-develop.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 target_repository:
   HTTPS: https://github.com/PyTorchLightning/lightning-flash.git
   # checkout a particular branch or a tag
-  checkout: master
+  checkout: ecosystem
```
Comment (Borda) [inline configs/PyTorchLightning/flash_pl-develop.yaml:4]: mind add note why this branch?

Comment (ethanwharris) [reply configs/PyTorchLightning/flash_pl-develop.yaml:4]: Just testing, was having some failures so trying to debug, will be switched back once done 😃

**Review by Borda — APPROVED** [review]

**Comment by Borda** [comment]
@ethanwharris seems it is still breaking...

**Comment by ethanwharris** [comment]
@Borda Should all be passing now 😃

**Comment by Borda** [comment]
> Should all be passing now

seems to be failing for an unrelated reason - pytest/flake8

**Review by Borda — APPROVED** [review]

**Pull request merged**